src/ceph-detect-init/test-requirements.txt

```
@@ -3,7 +3,7 @@ discover
 fixtures>=0.3.14
 python-subunit
 testrepository>=0.0.17
-testtools>=0.9.32
+testtools>=0.9.32,<=2.3.0
```

probably a better fix is something like
```suggestion
testtools>=0.9.32,<=2.3.0; python_version < 3.5
testtools>=2.4.0; python_version >= 3.5
```

also, i don't think ceph-disk qa suite exercises the tox based test of ceph-detect-init. it is performed by  "make-check" run though.

but unfortunately, all of our "ceph-pull-requests" jenkins job is always scheduled to bionic slaves which do not offer python3.4 -- bionic comes with python2.7 and python3.6. these python versions are supported by testtools 2.4, see https://pypi.org/project/testtools/2.4.0/ .

since "testtools>=0.9.32,<=2.3.0" works for python2.7, python3.4, python3.5 and python3.6. i am fine as it is. with or without the suggested change.

---

@tchaikov I tried that first and make-check failed (iirc. duplicate testtools entry or something like that). So having 2 entries does not work with the used pip version.
Given that this is only for mimic, I think the current approach is okish.